### ISSUE TITLE
QRC example is broken - Python's sys module only partially imported

### ISSUE THREAD

**Issue by drs251** [thp/pyotherside#84]
I'm trying to learn how to use pyotherside by running the included examples, and I have the following problem with the QRC example: 

```
"PyOtherSide error: Traceback (most recent call last):\n\n  File \"qrc:/below/qrc_example_below.py\", line 5, in <module>\n\nAttributeError: module 'sys' has no attribute 'path'\n"
Unhandled PyOtherSide error: Cannot import module: qrc_example_below (Traceback (most recent call last):

  File "qrc:/below/qrc_example_below.py", line 5, in <module>
```

This is quite weird, since the same call to `sys.path` works fine in qrc_example.qml. When I run `print(dir(sys))` in both Python files, it appears that some parts are missing in qrc_example_below.py, although they're using the same version of Python3 (which I also checked). I tried on Windows 10 with MSVC 2015 and Qt 5.9.1 and 5.6.2 and Anaconda 4.4.0, as well as on Ubuntu with GCC and the included version of Python3. Any ideas what might be happening?

**Comment by thp** [comment]
The issue was that we were decref'ing an internal Python object when we shouldn't have. Fixed in d1e3290.

**Comment by thp** [comment]
Fix released as version 1.5.3.
